engine/model.go

```diff
@@ -85,9 +85,40 @@ func (a *Address) Equals(o Address) bool {
 	return a.Network == o.Network && a.Address == o.Address
 }
 
+// Sets up OCSP stapling
+type OCSPSettings struct {
+	Enabled            bool
+	Period             string
+	Responders         []string // optional responder
+	SkipSignatureCheck bool
+}
+
+func (o *OCSPSettings) RefreshPeriod() (time.Duration, error) {
+	if o.Period == "" {
+		return time.Hour, nil
+	}
+	return time.ParseDuration(o.Period)
+}
+
+func (o *OCSPSettings) Equals(other *OCSPSettings) bool {
+	if o.Period != other.Period {
+		return false
+	}
+	if (len(o.Responders) == 0 || len(other.Responders) == 0) || len(o.Responders) != len(other.Responders) {
```

Comment:
I'm surprised two `OCSPSettings` structs are considered unequal if they both have 0 responders. Why is that?